core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -46,16 +46,16 @@ class ListConsumerGroupTest extends ConsumerGroupCommandTest {
     }, s"Expected --list to show groups $expectedGroups, but found $foundGroups.")
   }
 
-  @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
-  def testListWithUnrecognizedNewConsumerOption(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testListWithUnrecognizedNewConsumerOption(quorum: String, groupProtocol: String): Unit = {
```

Comment:
Does it bring any value to test all the combinations in this case?

Reply:
https://github.com/apache/kafka/pull/15150/files/7d9fb7d6587ffb114a67290653992720d85965b0#r1459837546

Reply:
I reiterate my question. The test verifies that `--new-consumer` is not accepted with `--list`. This is completely unrelated to zk vs kraft, old coordinator vs new coordinator, old consumer vs new consumer so all the combinations that we test here are not relevant. Would it make sense to keep it as it was before?